app/js/app.js

```diff
@@ -16,6 +16,7 @@ angular.module("FM", [
     "FM.sockets",
     "FM.loadingScreen",
     "ngRoute",
+    "notification",
```

Comment:
rather than injecting the library into app, instead inject it into just the modules of our app that need it e.g. `angular.module("FM.player.PlayerCtrl", ["notification"])`